extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/indexing/error/NotEnoughTemporaryStorageFault.java

```diff
@@ -41,7 +41,8 @@ public NotEnoughTemporaryStorageFault(
   {
     super(
         CODE,
-        "Not enough temporary storage space. Requires at least %,d bytes. (configured = %,d bytes)",
+        "Not enough temporary storage space for intermediate files. Requires at least %,d bytes. (configured = %,d bytes). Increase the limit by increasing tmpStorageBytesPerTask or "
+        + "disable durable storage to not use disk for temporary intermediate files.",
```

Comment:
```suggestion
        + "disable durable storage by setting the context parameter blah blah blah.",
```
Something like this.

Reply:
Changed